app/views/exchanges/index.html.erb

```diff
@@ -0,0 +1,22 @@
+<p style="color: green"><%= notice %></p>
+<div class="c-container">
+ <div class="header">
+  <%= link_to "Back", user_categories_path(current_user), class: "btn" %>
+  <h3 class="title">TRANSACTION</h3>
+  <div class="sign-out">
+  <%= link_to 'Sign Out', destroy_user_session_path, data: { turbo_method: :delete }, class: "btn" %>
+  </div>
+  </div>
+ 
+<div id="exchanges">
+<% total_amount = @exchanges.sum(&:amount) %>
+<div  class="total-amount"><h3>Total amount: $<%= total_amount %></h3></div>
+  <% @exchanges.each do |exchange| %>
+    <%= render exchange %>
+     
+  <% end %>
+</div>
+<div class="new">
```

Comment:
- [ ] As I said above, the same applies on the transactions page, I would suggest that you make the `Add transaction` button align at the bottom of the page 🙏🏾 
<img width="384" alt="Screen Shot 2023-09-16 at 18 38 27" src="https://github.com/isume295/Budget-app/assets/90222110/1396c998-cc76-483e-b51a-76491cf3732f">
<img width="268" alt="Screen Shot 2023-09-16 at 17 29 12" src="https://github.com/isume295/Budget-app/assets/90222110/b4e46b90-3351-41c1-8ca1-8d8dce4144cb">